render.go

```diff
@@ -247,6 +247,12 @@ func (r *Render) compileTemplatesFromAsset() {
 	}
 }
 
+// Wrapper around template.Lookup(). Returns the template with the given
```

Comment:
Run `golint` over this file... exported functions need a comment that starts with the function name.